tests/cli/test_run.py

```diff
@@ -57,6 +57,7 @@ def test_main_help(capsys, options):
         '--theme THEME, -t THEME',
         '-h, --help',
         '-d, --debug',
+        '--list-themes',
```

Comment:
This commit can be squashed with the previous one.

Reply:
Squashed the two commits into one. The next one will include the formatting and the other changes which you were talking about.